types/graphql-fields/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
   "dependencies": {
-    "graphql": "^15.3.0"
+    "graphql": "^16.6.0"
```

Comment:
Just to check, is there a reason to even pin a version here? Some packages just set `"*"` which might work well in this case to say "it doesn't matter".

Reply:
I can't think of a reason why it would matter, but I'm not an SME on this.

Reply:
The reason it might matter is that this will only be a `patch` bump to `@types/graphql-fields`, so those who are using an old revision of `graphql` will get double deps and potential type mismatches.

Reply:
Ah, fair.  I can unpin it; nothing I need this for would have issues with this being unpinned.

Reply:
Normally I'd say this should match peerDeps from the package, but, they didn't set any!